Makefile

```diff
@@ -319,12 +321,12 @@ packagedir: clean embed-download macos-release-binary packaging/vfkit-$(GOARCH).
 	mkdir -p packaging/root/Applications
 	tar -C packaging/root/Applications -xvzf $(TRAY_RELEASE)
 	rm packaging/tmp/crc-tray-macos.tar.gz
-	mv packaging/root/Applications/crc-tray-darwin-x64/crc-tray.app packaging/root/Applications/Red\ Hat\ OpenShift\ Local.app
-	rm -fr packaging/root/Applications/crc-tray-darwin-x64
+	mv packaging/root/Applications/crc-tray-darwin-$(TRAY_ARCH)/crc-tray.app packaging/root/Applications/Red\ Hat\ OpenShift\ Local.app
+	rm -fr packaging/root/Applications/crc-tray-darwin-$(TRAY_ARCH)
```

Comment:
currently we are not doing arm builds for the tray releases, maybe we can depend on rosetta to do the right thing here for the initial release, as we are also not using arm builds of `admin-helper`

am not able to build the pkg after this commit, since we don't have the tray arm bits:
```
mv packaging/root/Applications/crc-tray-darwin-arm64/crc-tray.app packaging/root/Applications/Red\ Hat\ OpenShift\ Local.app
mv: rename packaging/root/Applications/crc-tray-darwin-arm64/crc-tray.app to packaging/root/Applications/Red Hat OpenShift Local.app: No such file or directory
make: *** [packagedir] Error 1
```

we need a target for generating arm `pkg` as well `$(BUILD_DIR)/macos-amd64/crc-macos-arm64.pkg`

Reply:
can we easily make arm builds of the tray? 
admin helper should not be that difficult

Reply:
I keep forgetting about `admin-helper` because the m1 can run the x86_64 binary, so it was not a blocking issue for m1 testing.